editoast/src/modelsv2/rolling_stock_model.rs

```diff
@@ -23,6 +23,9 @@ use validator::ValidationErrors;
 
 use crate::modelsv2::prelude::*;
 
+pub mod schedules_from_rolling_stock;
```

Comment:
Is there anything else in the module that is exposed. I believe you can reduce the API here.

```suggestion
mod schedules_from_rolling_stock;
```